modin/pandas/dataframe.py

```diff
@@ -2700,6 +2700,117 @@ def style(df):
 
         return self._default_to_pandas(style)
 
+    def _build_batch_pipeline(self, func: Callable, axis: int, **kwargs):
+        """
+        Create a batched pipeline.
+
+        Parameters
+        ----------
+        func : Callable
+            The first function in the pipeline.
+        axis : int
+            The axis over which this pipeline will operate
+        **kwargs
+            Arguments to pass to the query object built for this function. For more information,
+            run `from modin.experimental.batch.pipeline import PandasQuery; PandasQuery.print_options()`
+
+        Returns
+        -------
+        DataFrame
+            A DataFrame with the pipeline.
+        """
+        from modin.experimental.batch.pipeline import PandasQueryPipeline
+        overwrite_existing = kwargs.pop("overwrite_existing", False)
+        if Engine.get() != "Ray":
+            ErrorMessage.not_implemented(
```

Comment:
Please skip coverage (`no_qa`) on these defensive coding components.

Reply:
I actually did add tests for this - would that help, or should I still skip it?